packages/wxt/src/core/package-managers/bun.ts

```diff
@@ -11,8 +11,8 @@ export const bun: WxtPackageManagerImpl = {
     if (options?.all) {
       args.push('--all');
     }
-    const { execa } = await import('execa');
-    const res = await execa('bun', args, { cwd: options?.cwd });
+    const { default: spawn } = await import('nano-spawn');
```

Comment:
Let's move this to a regular import as well, was doing dynamic because execa wasn't ESM before. Not sure about now, but nano-spawn is ESM-only.

Reply:
Will change